tests/compliance_operator_test.go

```diff
@@ -20,8 +20,8 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-var (
-	namespace = "openshift-compliance"
+const (
+	coNamespace = "openshift-compliance"
```

Comment:
coNamespace is from compliance or from the second namespace, if the first then let's name it `compliacneNamespace`

Reply:
`co` stands for **C**ompliance **O**perator, I've seen this name used elsewhere.